### PR TITLE
#625/remove vacancy-message

### PR DESCRIPTION
**Pull request by tomlovesgithub**
- Removed `vacancy-message.vue` and references in router. 
- Made links that would direct to that page just go through to `eligibilty-check.vue` (the citizenship question).
I think this covers everything needed to remove this and patch the journey.

There was no script of any kind in the vacancy message I'm sure just removing it will have no unintended side effects.

### CODE REVIEW

File: src/router.js [hunk 1]
```diff
@@ -13,7 +13,7 @@ import ResetPassword from '@/views/ResetPassword';
 import Vacancies from '@/views/Vacancies';
 import Vacancy from '@/views/Vacancy';
 import VacancyDetails from '@/views/Vacancy/VacancyDetails';
-import VacancyMessage from '@/views/Vacancy/VacancyMessage';
+// import VacancyMessage from '@/views/Vacancy/VacancyMessage';
```
Comment (warrensearle) [inline src/router.js:16]: Can remove this line rather than comment out

**Review by warrensearle — CHANGES_REQUESTED** [review]
Teeny tiny tweak

**Review by warrensearle — APPROVED** [review]
Looks good to me 👍

File: src/router.js (hunk 1 above)
Comment (tomlovesgithub) [reply src/router.js]: 🤷‍♂️

**Pull request merged**